src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx

```diff
@@ -126,14 +134,37 @@ const ValidatorsList = ({ onCheckboxClickHandler, selectedValidators, selectUnse
   //   });
   // };
 
+  const copyToClipboard = (publicKey: string) => {
+    navigator.clipboard.writeText(publicKey);
+    notificationsStore.showMessage('Copied to clipboard.', 'success');
+  };
+
+  const openBeaconcha = (publicKey: string) => {
+    GoogleTagManager.getInstance().sendEvent({
+      category: 'external_link',
+      action: 'click',
+      label: 'Open Beaconcha',
+    });
+    window.open(`${ENV().BEACONCHA_URL}/validator/${publicKey}`);
+  };
+
+  const openExplorer = (publicKey: string) => {
+    GoogleTagManager.getInstance().sendEvent({
```

Comment:
Why do we need to add a GoogleTagManager event here? Does anyone look at the google dasboard?

Reply:
@IlyaVi we need it?

Reply:
removed for now, will work on this later